frontend/public/models/yaml-templates.ts

```diff
@@ -797,4 +797,17 @@ spec:
   machineSelector:
     matchLabels:
       node-role.kubernetes.io/master: ""
+`).setIn([referenceForModel(k8sModels.MachineAutoscalerModel), 'default'], `
+apiVersion: "autoscaling.openshift.io/v1beta1"
+kind: "MachineAutoscaler"
+metadata:
+  name: "worker-us-east-1a"
+  namespace: "openshift-cluster-api"
```

Comment:
This is the wrong namespace. It should be: `openshift-machine-api`

Reply:
Thanks, I updated it. It gets replaced anyway at runtime anyway FWIW. This is a template for the YAML editor, which defaults the namespace to the current namespace. It's not used for the dialog.

Reply:
Cool. Thought that might be the case, but wanted to mention it anyway. Thanks!